db/create-db.sql

```diff
@@ -71,30 +71,34 @@ CREATE TABLE emissions(
 -- ------------------------------------------------------------------------------------
 
 -- Have started to populate but need to get better carbon emissions figures.
+-- LJ Notes :
+-- The API returns driving, walking, bicycling and transit. 
+-- Transit would take into consideration just bus and private vehicle can take into consideration a diesel car or an elec vehicle.
+-- Private vehicle can take into consideration if travelling with multiple people. Emission sharing i.e Emission/no.of people travelling
 
 INSERT INTO vehicles
 (vehicle_name, carb_emit_km)
 VALUES
 -- manual vehicles
 ('foot', 0),
-('bicycle', 21),
+('bicycle', 21),   
 -- electric vehicles
 ('elec_bike', 22),
 ('elec_scooter', 0),
-('elec_car', 0),
+('elec_car', 0), 
 -- private fossil fuel vehicles
 ('moped', 0),
-('motorbike', 115),
-('hybrid_car', 0),
-('petrol_car', 271),
-('diesel_car', 0),
+('motorbike', 113.37),  -- https://www.thrustcarbon.com/insights/how-to-calculate-motorbike-co2-emissions
+('hybrid_car', 40),     -- http://co2cars.apps.eea.europa.eu/?source=%7B%22track_total_hits%22%3Atrue%2C%22query%22%3A%7B%22bool%22%3A%7B%22must%22%3A%5B%7B%22constant_score%22%3A%7B%22filter%22%3A%7B%22bool%22%3A%7B%22must%22%3A%5B%7B%22bool%22%3A%7B%22should%22%3A%5B%7B%22term%22%3A%7B%22year%22%3A2020%7D%7D%5D%7D%7D%2C%7B%22bool%22%3A%7B%22should%22%3A%5B%7B%22term%22%3A%7B%22scStatus%22%3A%22Provisional%22%7D%7D%5D%7D%7D%5D%7D%7D%7D%7D%5D%7D%7D%2C%22display_type%22%3A%22tabular%22%7D
+('petrol_car', 141.9),  
+('diesel_car', 151.3),
 -- public transport vehicles
-('bus', 101),
+('bus', 13.7),          -- avg: 822g/km, avg bus capacity: 60.https://www.carbonindependent.org/20.html#:~:text=Some%20key%20points%20are%3A,buses%20is%20822%20g%20%2F%20km
 ('coach', 27),
-('train', 0),
-('tram', 0),
-('private taxi', 0),
-('shared taxi', 0);
+-- ('train', 0),            Probably wouldn't need them as API wouldn't return time for the transit
+-- ('tram', 0),
+('private taxi', 50),    -- ZEC Vehicles. https://tfl.gov.uk/info-for/taxis-and-private-hire/emissions-standards-for-phvs#:~:text=Zero%20Emission%20Capable%20(ZEC)%20vehicles,-To%20meet%20the&text=Emit%20no%20more%20than%2050g,miles%20(16.093%20km)%3B%20or
+-- ('shared taxi', 0);     need a function to divide private taxi/no. of people
 
 
 -- RESEARCH NOTES:
```

Comment:
Just been thinking and it might be best we get all the data from a single source to minimise the errors within the data. By getting our data from different sources we are introducing more uncertainty as different sources might be calculating things differently and using data from different years. If we use just one source, any margin of error in one mode of transport would hopefully be relative to the others.

I propose we use the [UK government stats](https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021). On the attached spreadsheet we would take data from the "passenger vehicles" and "business travel- land" tabs.